ingest/indexer/service/client/pubsub_client.go

```diff
@@ -23,6 +23,10 @@ type PubSubClient struct {
 	pubsubClient             *pubsub.Client
 }
 
+const (
+	MAX_PUBLISH_DELAY = 4 * time.Second
```

Comment:
what's the reason we don't add this to the config options?

Reply:
Right! Why didn't I do that ? 😄 
Done and tested by `osmosisd init test` to make sure it generates the right content and parses it.